web/cobrands/sass/_base.scss

```diff
@@ -1957,6 +1957,85 @@ html.js #map .noscript {
   @include svg-background-image("/cobrands/fixmystreet/images/map-tools");
 }
 
+// Map instructions for keyboard
+.keyboard-intructions-wrapper {
+  display:flex;
+  position: absolute;
+  padding: 10px 25px;
+  padding-top: 14px;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  bottom: 35px;
+  left: 50%;
+  transform: translateX(-50%);
+
+  span {
+    display: inline-block;
+    margin-left: 20px;
+    font-weight: 600;
+  }
+
+  &.basic {
+    opacity: 0;
+  }
+
+  @media only screen and (max-width:48em) {
+    display: none;
+  }
+}
+
+.keyboard-key-icon {
+  display: inline-block;
+  border-radius: 4px;
+  width: 30px;
+  height: 0;
+  padding-top: 22px;
+  overflow: hidden;
+  background-color: #ECEBEB;
+  box-shadow: 0px 3px 0px rgba(0, 0, 0, 0.3);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &.zoom-in {
+    background-image: url("data:image/svg+xml,%3Csvg fill='none' height='14' viewBox='0 0 13 14' width='13' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='%23222'%3E%3Crect height='13' rx='.3' width='3.25' x='4.875' y='.5'/%3E%3Crect height='13' rx='.3' transform='matrix(0 -1 1 0 -8.625 8.625)' width='3.25' y='8.625'/%3E%3C/g%3E%3C/svg%3E");
```

Comment:
My instinct here would be to either:

1. Use the `svg-background-image()` mixin for these too; or…
2. Embed the SVG markup directly into the HTML template

Either way would be a big improvement over maintaining escaped SVG markup in a `background-image: url()` definition like this.